docs/building_images_with_bazel.md

```diff
@@ -0,0 +1,300 @@
+---
+tilt: Building Images with Bazel
+layout: docs
+---
+
+# Building Images with Bazel
+
+There's nothing more frustrating than using a tool that works perfectly except for _one_ small thing. Even if the tool is open source it can be a daunting task to get a change accepted to it.
+
+We recognize that everyone's set up is different and that we cannot foresee all possible features and use cases that folks will need. Software is complicated! As a result we designed Tilt to be customizable from the beginning. If there's functionality you wish Tilt had you can often add it yourself, right in your Tiltfile, no external pull requests necessary!
```

Comment:
```suggestion
We recognize that everyone's set up is different and that we cannot foresee all possible features and use cases that folks will need. Software is complicated! As a result we designed Tilt to be customizable from the beginning. If there's functionality you wish Tilt had you can often add it yourself, right in your Tiltfile, no code changes or pull requests necessary!
```

This might be nitty, but the "external" in "external pull requests" rubbed me a bit weirdly - it's framed relative to tilt devs, who are not the intended audience.